pilot/pkg/networking/core/v1alpha3/httproute.go

```diff
@@ -250,10 +250,19 @@ func (configgen *ConfigGeneratorImpl) buildSidecarOutboundHTTPRouteConfig(env *m
 
 	out := &xdsapi.RouteConfiguration{
 		Name:             routeName,
-		VirtualHosts:     virtualHosts,
 		ValidateClusters: proto.BoolFalse,
 	}
 
+	//BAVERY_TODO: Find a better way to handle these environment variables than parsing each time
```

Comment:
`env.RegisterBoolVar` will do that

Reply:
Does it work for node metadata too ? It's not env.